### PR TITLE
Add config from Funnelback york-uni-courses

### PR DESCRIPTION
**Pull request by ChristianBeddows**

### CODE REVIEW

**Review by RichardAllum — COMMENTED** [review]
If we want the courses collection to only be accessible via the Courses API, should we delete `simple.form` and `simple.ftl`? I'm not sure if there are any benefits to having a search form page for this collection, but it could lead to increased maintenance work in future.

**Review by ncn1 — COMMENTED** [review]
I think this is a long way off what we want from this collection at the moment, so I question if merging it into master at this point is appropriate (unless that's the only way GitHub allows for code review?). It doesn't yet contain the fields we require for the API and History and French comes at the top of the results when 'history' is searched for, it says there are no fully matched results when the york-uni-web collection returns 197 fully matched results for the same search etc.

File: config/collection.cfg [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#
+# Filename: /opt/funnelback/conf/york-uni-courses/collection.cfg
+#
+admin_email=marketing-support@york.ac.uk
```
Comment (ce666-york) [inline config/collection.cfg:4]: This makes me wonder if there are any notifications from Funnelback relevant to us that we could hook into

**Comment by ce666-york** [comment]
> 
> 
> If we want the courses collection to only be accessible via the Courses API, should we delete `simple.form` and `simple.ftl`? I'm not sure if there are any benefits to having a search form page for this collection, but it could lead to increased maintenance work in future.

I agree if we don't need it let's remove it, if we can

**Comment by ce666-york** [comment]
> 
> 
> I think this is a long way off what we want from this collection at the moment, so I question if merging it into master at this point is appropriate (unless that's the only way GitHub allows for code review?). It doesn't yet contain the fields we require for the API and History and French comes at the top of the results when 'history' is searched for, it says there are no fully matched results when the york-uni-web collection returns 197 fully matched results for the same search etc.

I wonder how much of that is down to best bets/manual tweaking vs the collection's configuration. Worth doing some investigation in Funnelback and maybe posing these questions to Digital Marketing

**Comment by ncn1** [comment]
> > If we want the courses collection to only be accessible via the Courses API, should we delete `simple.form` and `simple.ftl`? I'm not sure if there are any benefits to having a search form page for this collection, but it could lead to increased maintenance work in future.
> 
> I agree if we don't need it let's remove it, if we can

I think we need to have a default template - and we certainly can't remove this until we've got everything working as required else we cannot do test searches.

**Review by ncn1 — APPROVED** [review]
The york-uni-courses collection now brings back the courses metadata and works with the course-search.ftl, and is correctly showing full matches. Best bets to get the most relevant results to the top of the results list can be added later.

**Review by ncn1 — APPROVED** [review]

**Pull request merged**